src/Table/web/TD.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../../theme';
+
+const TD = styled.td`
```

Comment:
How about renaming to `Cell`?
`Table.Cell` makes more sense.

Reply:
Makes sense! Was trying to keep naming in sync with html table tags.